pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -3,8 +3,21 @@ title: Included in the accessibility tree
 key: included-in-the-accessibility-tree
 ---
 
-Elements included in the accessibility tree of platform specific accessibility APIs. Elements in the accessibility tree are exposed to assistive technologies, allowing users to interact with the elements in a way that meet the requirements of the individual user.
+Elements included in the accessibility tree of platform specific accessibility APIs. 
 
-The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are [included in the accessibility tree][] can be found in the [HTML accessibility API mappings](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings](https://www.w3.org/TR/svg-aam/).
+Elements in the accessibility tree are exposed to assistive technologies, allowing users to interact with the elements in a way that meet the requirements of the individual user.
+
+An element is included in the accessibility tree if it is a DOM node in the flattened DOM tree that meets the following criteria:
+
+1. It isn't hidden through WAI-ARIA or CSS, and
```

Comment:
Should we clarify aria-hidden, display:none or visibility:hidden? There are only 3 options here so maybe we can just include them.